src/KeyVault/KeyVault/Commands/Key/GetAzKeyVaultRandomNumber.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.Azure.Commands.KeyVault.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.KeyVault.Commands.Key
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzurePrefix + "KeyVaultRandomNumber", DefaultParameterSetName = GetByHsmNameParameterSet)]
+    [OutputType(typeof(string), typeof(byte[]))]
```

Comment:
```suggestion
    [OutputType(typeof(string), typeof(byte))]
```
Do not use array or list as the output type. See https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#output-type